db/schema.rb

```diff
@@ -56,6 +56,18 @@
     t.index ["visit_token"], name: "index_ahoy_visits_on_visit_token", unique: true
   end
 
```

Comment:
I somehow accidentally added this table and having difficulty getting the repo to forget it (I was doing a spike on adding translations to a database table).  Can you remove this? @peterdavidhamilton